docs/openshift_libvirt.md

```diff
@@ -63,3 +63,14 @@ $ virsh net-start openshift
 ```
 $ virsh net-autostart openshift
 ```
+
```

Comment:
Thanks for updating the docs as well! Can we move this paragraph to the front page, just after the "Adding Cluster Nodes" section? And probably change the title to "Deleting OpenShift" so it matches with the existing section "Installing OpenShift"?

Reply:
Moved the paragraph to the front page.